### PR TITLE
Asecsion ui

### PR DESCRIPTION
**Pull request by sshumake**
@ardasgup @pbathia @avijra @renujhamtani 

Hey this is a PR thats only 90%(it works but missing some polish) that I intend to finish tomorrow.  However it has a temporary fix for a UDS change.  If you would like to merge please merge, other wise check out the fix and push a separate PR.

### CODE REVIEW

File: app/ascension/services/caseDetailsService.js [hunk 1]
```diff
@@ -189,7 +191,9 @@ angular.module('RedhatAccess.ascension').service('CaseDetailsService', [
             var self = this;
             ssoUserName = securityService.loginStatus.authedUser.sso_username;
             var userUql = UQL.cond('SSO','is',"\""+ ssoUserName + "\"");
-            udsService.user.get(userUql).then(angular.bind(this, function (user){
+            udsService.user.get(userUql).then(angular.bind(this, function (userArr){
+                //TODO should search fur contact first then default to user
```
Comment (sshumake) [inline app/ascension/services/caseDetailsService.js]: @ardasgup @pbathia @avijra @renujhamtani 
Temporary fix

Comment (ardasgup) [inline app/ascension/services/caseDetailsService.js]: @sshumake i feel we should check out just this fix and wait to merge the rest. Although rest looks good.

Comment (pbathia) [inline app/ascension/services/caseDetailsService.js]: This needs to be reverted, as Adam is making changes for not to return array inside array.

File: test/spec/ascension/ascension.services.test.js [hunk 2]
```diff
@@ -54,7 +54,8 @@ describe('Ascension Services', function () {
         });
         it('should have a method for extracting routing roles for given user', function () {
             expect(caseDetailsService.extractRoutingRoles).toBeDefined();
-            var result = caseDetailsService.extractRoutingRoles(mockUDSDataService.mockUser);
+            var result = caseDetailsService.extractRoutingRoles(mockUDSDataService.mockUser[0]);
+            console.log(result);
```
Comment (pbathia) [inline test/spec/ascension/ascension.services.test.js]: this needs to be reverted.

File: test/mocks/mockUDSService.js [hunk 3]
```diff
@@ -59,40 +59,84 @@ angular.module('RedhatAccess.mockUDS', [])
...
+                                    "sso":[
+                                       "rhn-support-sshumake"
+                                    ]
+                                 },
+                                 "resourceReliability":"Fresh",
+                                 "resourceType":"Contact",
+                                 "externalModelId":"003A0000016KcfLIAS"
+                              }
+                           ]
+                        ]
```
Comment (pbathia) [inline test/mocks/mockUDSService.js]: This also needs to be reverted, as Adam is making changes /user endpoint

**Comment by pbathia** [comment]
Rest all UI changes looks fine.

**Pull request merged**